### PR TITLE
Adding get_giphy

### PR DESCRIPTION
**Pull request by sdepew**
Adding get_giphy since we use it all over the place.

### CODE REVIEW

File: bot/utils/__init__.py [hunk 1]
```diff
@@ -88,3 +92,29 @@ def get_lat_lng(input=[], strip_terms=[]):
         logger.error(
             "HTTP Error from Google.\nError Code: {}\n Error: {}".format(request.status_code, request.content))
         return False
+
```
Comment (LEXmono) [inline bot/utils/__init__.py]: One more blank line please.

File: bot/utils/__init__.py [hunk 2]
```diff
@@ -88,3 +92,29 @@ def get_lat_lng(input=[], strip_terms=[]):
...
+        'lang': 'en',
+        'offset': random.choice([x for x in range(0, 7)])
+    }
+    request = requests.get(giphy_url, params=params)
+    if request.ok:
+        gif_url = random.choice(request.json()['data'])['images']['downsized']['url']
+        logger.debug("Successfully retrieved Gif: {}".format(gif_url))
+        response = {
+            "text": "",
+            "attachments": [
```
Comment (LEXmono) [inline bot/utils/__init__.py]: Do we want to ALWAYS return a GIF as an attachment, or do we want to leave that up to the individual functions? I'm okay with either. Just want to put the thought out there.

Comment (sdepew) [reply bot/utils/__init__.py]: I say always return the attachment

File: bot/utils/__init__.py (hunk 1 above)
Comment (sdepew) [reply bot/utils/__init__.py]: done

**Pull request merged**